index.js

```diff
@@ -0,0 +1,24 @@
+import tokenize from '@warp-ds/tokenizer';
+
+import { init, brandToName, downloadReleaseFile, geneateFinalCss, getBrandModes, processHexCss, processRGBCss } from './utils.js';
+
+init();
```

Comment:
Again, might be missing something that was already mentioned in some discussion, but I can't see that this script is run anywhere. This file lives on the root dir as if it's a common script for all stylesheets living in the css repo, but it's in fact only a script generating tokens CSS. Should it then replace the /tokens/index.js file?

Reply:
Its now running via `pnpm build:tokens`